packages/wallet-stamper/src/__tests__/wallet-interfaces.ts

```diff
@@ -51,7 +51,9 @@ export class MockEvmWallet implements EvmWalletInterface {
     return signature;
   }
 
-  async recoverPublicKey(message: string, signature: string): Promise<string> {
+  async getPublicKey(): Promise<string> {
+    const message = "hello from TKHQ!";
```

Comment:
I like this message! :D maybe we should also use `hello from TKHQ!` in the sdk-browser README: `packages/sdk-browser/README.md`